Avatar/avatar.css

```diff
@@ -0,0 +1,112 @@
+*,
+*::before,
+*::after {
+  box-sizing: border-box;
+  margin: 0;
+  padding: 0;
+}
+
+:root{
+    --gray:#3e4152;
+    --text-color:#007FFF;
+    --blue-bg:#023e8a;
+    --bg-code:#FAFAFA;
+    --colour-txt:white;
+}
```

Comment:
Suggestion: 
Please follow hex code for consistency and rename your variables for better readability.